features/support/sauce_driver.rb

```diff
@@ -0,0 +1,19 @@
+module SauceDriver
+  class << self
+    def config
```

Comment:
We already get this functionality from the `config` gem.  Check my comment above about using the `Settings.saucelabs.something`.